<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../App_res.qrc">
    <normaloff>:/Images/Images/Icon.png</normaloff>:/Images/Images/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="activationOrder">
       <enum>QMdiArea::CreationOrder</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionReading"/>
    <addaction name="actionRecording"/>
    <addaction name="separator"/>
    <addaction name="actionControl_Panel"/>
    <addaction name="actionOptions"/>
    <addaction name="actionPlugin_Manager"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
   <addaction name="actionControl_Panel"/>
   <addaction name="separator"/>
   <addaction name="actionReading"/>
   <addaction name="actionRecording"/>
   <addaction name="separator"/>
   <addaction name="actionCascade"/>
   <addaction name="actionTile"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/folder.png</normaloff>:/Images/Images/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/disk.png</normaloff>:/Images/Images/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/save_as.png</normaloff>:/Images/Images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/page_add.png</normaloff>:/Images/Images/page_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/printer.png</normaloff>:/Images/Images/printer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/door_in.png</normaloff>:/Images/Images/door_in.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionReading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/run.png</normaloff>:/Images/Images/run.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reading</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/control_pause_record.png</normaloff>:/Images/Images/control_pause_record.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;cording</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionControl_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/speedometer.png</normaloff>:/Images/Images/speedometer.png</iconset>
   </property>
   <property name="text">
    <string>Control &amp;Panel</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/screwdriver.png</normaloff>:/Images/Images/screwdriver.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPlugin_Manager">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/plugin.png</normaloff>:/Images/Images/plugin.png</iconset>
   </property>
   <property name="text">
    <string>Plugin &amp;Manager</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/application_cascade.png</normaloff>:/Images/Images/application_cascade.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/application_tile_horizontal.png</normaloff>:/Images/Images/application_tile_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/help.png</normaloff>:/Images/Images/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/update.png</normaloff>:/Images/Images/update.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Update</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/info_rhombus.png</normaloff>:/Images/Images/info_rhombus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../App_res.qrc">
     <normaloff>:/Images/Images/icon64.png</normaloff>:/Images/Images/icon64.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../App_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionTile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
